packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -211,6 +218,12 @@ class HeapTreeViewState extends State<HeapTree>
       ));
     });
 
+    addAutoDisposeListener(controller.monitorAllocationsChanged, () {
```

Comment:
calls to the controller shouldn't be wrapped in a call to setState. You can cleanup by listening to monitorAllocationsChanged directly from the controller
and listening for the libraries changing from directly in the Widget.

Reply:
The name was poorly choosen should be monitorAllocationsNotifier (its a ValueNotifier).